src/documents/forms/DocumentTitleValidationIndicators.js

```diff
@@ -38,21 +38,29 @@ const DocumentTitleValidationIndicators = ({errors}) => {
       <List.Item>
         <Icon
           name={
-            (errors.file_name && errors.file_name.file_ext) ||
-            (errors.file_name && errors.file_name.upload_similarity)
+            errors.file_name && errors.file_name.upload_similarity
               ? 'x'
               : 'check'
           }
           color={
-            (errors.file_name && errors.file_name.file_ext) ||
-            (errors.file_name && errors.file_name.upload_similarity)
+            errors.file_name && errors.file_name.upload_similarity
               ? 'red'
               : 'black'
           }
         />
         Name your document different than the uploaded file to help give us more
         context.
       </List.Item>
+      <List.Item>
+        <Icon
+          name={errors.file_name && errors.file_name.file_ext ? 'x' : 'check'}
+          color={
+            errors.file_name && errors.file_name.file_ext ? 'red' : 'black'
+          }
+        />
+        Avoid using file extensions, the file name and its extension are already
```

Comment:
I think we could combine this with the `   Name your document different than the uploaded file ..` maybe just need some better wording 

@allisonheath thoughts?